paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -337,6 +337,18 @@ public class CoreOptions implements Serializable {
                     .withDescription(
                             "Cache size for reading manifest files for write initialization.");
 
+    public static final ConfigOption<Integer> FILE_INDEX_BLOOM_FILTER_ITEMS =
+            key("file-index.bloom-filter-items")
+                    .intType()
+                    .defaultValue(150)
```

Comment:
default value 150?

Reply:
default 150, and default store in data meta size is 500 B, then we could store 3 column index in data meta.

Reply:
![image](https://github.com/apache/paimon/assets/41894543/163c5b63-0559-427d-949e-3d31b2dd8f2e)